PixelDungeon/src/main/res/values/strings.xml

```diff
@@ -184,6 +184,8 @@
 
     <!-- STRINGS OF WndJournal CLASS -->
     <string name="WndJournal_Title">Journal</string>
+    <string name="WndJournal_Levels">Levels</string>
```

Comment:
Can you please add all new strings to strings_remix_next.xml?
This is sound a bit strange I know, but it let me feel safe while uploading new file to transifex.

Reply:
Should I move them or should they be in both files?

Reply:
Please move them.

Reply:
Done. Moved to the `strings_remix_next.xml`, but only the 2 new strings I added.